modules/alloy/src/test/ts/browser/position/SelectionInDocPositionTest.ts

```diff
@@ -10,6 +11,26 @@ import * as ChainUtils from 'ephox/alloy/test/ChainUtils';
 import * as PositionTestUtils from 'ephox/alloy/test/PositionTestUtils';
 import * as Sinks from 'ephox/alloy/test/Sinks';
 
+const toDomRange = (cursorRange: Cursors.CursorRange): Range => {
+  const doc = Traverse.owner(cursorRange.start);
+  const range = doc.dom.createRange();
+  range.setStart(cursorRange.start.dom, cursorRange.soffset);
+  range.setEnd(cursorRange.finish.dom, cursorRange.foffset);
+  return range;
+};
+
+const cAssertBelowSelection = NamedChain.bundle((data: any) => {
+  const root = data.inline.element;
+  const popup = data.popup;
+  const path = data.path;
+  const range = toDomRange(Cursors.calculate(root, path));
+
+  const selectionBox = range.getBoundingClientRect();
+  const popupBox = popup.element.dom.getBoundingClientRect();
+  assert.approximately(popupBox.y, selectionBox.bottom, 5);
```

Comment:
This doesn't assert it's below it, only that it's within 5px of it. So we'll need both assertions here.